### PR TITLE
Fixed the sprint speeds of Desktop vs HMD

### PR DESCRIPTION
**Pull request by orenh1**
Avatars have different sprint speeds in HMD and Desktop modes. The code that sets the speed had the values reversed.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Can someone from High Fidelity approve this PR?

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22066/HighFidelity-Beta-PR16390-bdb0085.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22066/HighFidelity-Beta-Interface-PR16390-bdb0085.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/22066/HighFidelity-Beta-PR16390-bdb0085.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/22066/HighFidelity-Beta-Interface-PR16390-bdb0085.exe

File: interface/src/avatar/MyAvatar.cpp [hunk 1]
```diff
@@ -5205,10 +5205,10 @@ bool MyAvatar::isReadyForPhysics() const {
 
 void MyAvatar::setSprintMode(bool sprint) {
     if (qApp->isHMDMode()) {
-        _walkSpeedScalar = sprint ? AVATAR_DESKTOP_SPRINT_SPEED_SCALAR : AVATAR_WALK_SPEED_SCALAR;
+        _walkSpeedScalar = sprint ? AVATAR_HMD_SPRINT_SPEED_SCALAR : AVATAR_WALK_SPEED_SCALAR;
     }
     else {
```
Comment (ZappoMan) [inline interface/src/avatar/MyAvatar.cpp]: I know you didn't introduce this line, but can you please repair the coding standard mismatch here

**Comment by DouglasWilcox** [comment]
Hi Oren, you can change that constant to 2.0f here:

libraries/shared/src/AvatarConstants.h

so both lines should now be 2.0f, like so:

static const float AVATAR_DESKTOP_SPRINT_SPEED_SCALAR = 2.0f;
static const float AVATAR_HMD_SPRINT_SPEED_SCALAR = 2.0f;

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22160/HighFidelity-Beta-PR16390-35082a7.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22160/HighFidelity-Beta-Interface-PR16390-35082a7.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/22160/HighFidelity-Beta-PR16390-35082a7.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/22160/HighFidelity-Beta-Interface-PR16390-35082a7.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22160/HighFidelity-Beta-PR16390-35082a7.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22160/HighFidelity-Quest-Beta-PR16390-35082a7.apk)

**Comment by orenh1** [comment]
I have implemented the requested changes.

**Comment by shanzzam** [comment]
For internal tracking https://highfidelity.atlassian.net/browse/DEV-2597

**Pull request merged**